src/components/todo/Todo.js

```diff
@@ -1,8 +1,19 @@
-import React, {Component} from 'react';
-import styled from 'styled-components';
+import React, { Component } from "react";
+import styled from "styled-components";
 
-import TodoInput from './TodoInput';
-import TodoCard from './TodoCard';
+import TodoInput from "./TodoInput";
+import TodoCard from "./TodoCard";
+
+const Div = styled.div`
```

Comment:
변수 네이밍의 핵심은 이 변수의 의도 및 역할을 잘 전달하는 것이에요!
Div라는 말 보다는 좀 더 이 변수의 특징을 반영하면 좋을 것 같아요.
예를 들자면 Wrapper라던가, Col 같은 이름이요. ColWrapper도 좋구요 ㅋㅋ

Reply:
TodoInput Component에서는 Row라는 네이밍을 쓰셨으니 여기도 Col로 통일하는게 좋을 것 같아용
Q) 그러면 Wrapper로써의 의미가 퇴색되는데 어떻게 하나요?
A) 사실 베스트는 이 방향입니다.
```suggestion
const Col = styled.div``; // 별도 파일 Col.js에서 선언 후 export
const Wrapper = styled(Col)` // Col을 import해 추가 Styling
```
이렇게 아주 기본적인 Component들을 별도로 쪼개서 관리하는 Component Sturcture 구성방식을 Atomic Design이라고 한답니다. 자세한 내용은 스터디때 소개해 드릴게요~
[참고할만한 글](https://github.com/danilowoz/react-atomic-design)